lighty-examples/lighty-community-restconf-dom-actions-app/README.md

```diff
@@ -0,0 +1,70 @@
+# Lighty RESTCONF DOM Actions Application
+This application provides RESTCONF north-bound interface with example of DOM action registration on the controller.
+
+This application starts:
+* Lighty Controller
+* OpenDaylight RESTCONF plugin
+* OpenDaylight Swagger servlet
+* NETCONF south-bound plugin
+
+![architecture](docs/restconf-netconf-controller-architecture.svg)
+
+This roughly translates to OpenDaylight feature set installed by karaf command:
+```
+feature:install odl-netconf-all
+```
+
+## Build and Run
+build the project: ```mvn clean install```
+
+### Start this demo example
+* build the project using ```mvn clean install```
+* go to target directory ```cd lighty-examples/lighty-community-restconf-dom-actions-app/target``` 
+* unzip example application bundle ```unzip  lighty-community-restconf-dom-actions-app-<VERSION>-bin.zip```
+* go to unzipped application directory ```cd lighty-community-restconf-dom-actions-app-<VERSION>```
+* start controller example controller application ```java -jar lighty-community-restconf-domo-actions-app-<VERSION>.jar``` 
+
+### Test example application
+Once example application has been started using command ```java -jar lighty-community-restconf-dom-actions-app-<VERSION>.jar``` 
+RESTCONF web interface is available at URL ```http://localhost:8888/restconf/*```
+
+##### URLs to start with
+* __POST__ ```http://127.0.0.1:8888/restconf/data/example-data-center:device/start```
+
+1. Use the payload like the following:
+```json
+{
+    "input": {
+        "start-at": "2021-09-09T16:20:00Z"
+    }
+}
+```
+
+2. You will receive output like the following:
+
+```json
+{
+    "example-data-center:output": {
+        "example-data-center:start-finished-at": "2021-09-09T16:20:00Z"
+    }
+}
+```
```

Comment:
please describe briefly what is happening here, what represents the payload and from where the output came from (lighty pass-through the input value prepared in `DeviceStartActionImpl`)